vrs_anvil/evidence.py

```diff
@@ -367,3 +164,54 @@ def fetch_by_vrs_ids(
     data = result.fetchall()
     conn.close()
     return data
+
+
+def get_vcf_row(
+    variant_id: str, vcf: VariantFile, index_path: str = None
+) -> VariantRecord:
+    """given a variant id and annotated VCF, get the associated VCF row
+
+    Args:
+        variant_id (str): VRS ID for the variant of interest
+        vcf (VariantFile): Pysam VariantFile
+        index_path (str, optional): Index used to speed up search for variant. Defaults to iterating through VCF.
+
+    Raises:
+        Exception: outputs if no index is found
+
+    Returns:
+        VariantRecord: A Pysam VariantRecord (VCF row)
+    """
+
+    assert "VRS_Allele_IDs" in vcf.header.info, (
```

Comment:
not sure if you're still getting around to a pass on this but just a reminder that `assert` statements are a little unreliable at runtime. This could probably be a ValueError or something more specific